wordpress-mobile/WordPress-FluxC-Android
# Shipping labels: Address verification

Fixes #1716.

This PR adds client support for the address verification endpoint.

**To test:**

1. Authenticate
2. Go to Woo -> Shipping labels
3. Select site with shipping labels
4. Tap on the Verify address button
5. Fill out the form with a valid address
6. Tap on the Verify address button
7. Notice a normalized address object is returned
8. Fill out an invalid address
9. Notice an error is displayed
